src/libraries/System.Security.Cryptography/tests/X509Certificates/ChainTests.cs

```diff
@@ -656,11 +645,14 @@ public static void BuildChain_FailOnlyApplicationPolicy()
                         X509ChainStatusFlags.NoError,
                         (a, status) => a | status.Status));
 
-                Assert.Equal(
-                    X509ChainStatusFlags.NotValidForUsage,
-                    holder.Chain.ChainElements[2].ChainElementStatus.Aggregate(
-                        X509ChainStatusFlags.NoError,
-                        (a, status) => a | status.Status));
+                if (!PlatformDetection.IsWindows || PlatformDetection.IsWindows10Version22000OrGreater)
```

Comment:
@bartonjs this one is interesting on the Windows 10 SP0 leg this test fails because the Application Policy violation `NotValidForUsage` does not get propagated to the root in the chain. I can't reproduce locally on Windows 11.

I don't know why it changed from one root to another.

Open to ideas on how we should handle this better for this test.